build.yaml

```diff
@@ -1551,3 +1551,42 @@ steps:
     - copy_files
     - ci_utils_image
     - build_hail
+ - kind: buildImage
+   name: hail_public_image
+   dockerFile: docker/hail/Dockerfile
+   contextPath: docker/hail/
+   publishAs: hail_public
+   dependsOn:
+    - build_hail
```

Comment:
does this work now? I thought you had to run dev deploy for it to work. I was expecting to see scopes set to dev only.

Reply:
The image build works fine, I just have no means to upload the resulting images to DockerHub.

As normal build steps, they are tested regularly and built on each master merge. Updating DockerHub is as easy as loading the deploy build and finding the gcr.io name for the latest built image.